net/knot-resolver/files/kresd.init

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh /etc/rc.common
+
+. /usr/share/resolver/functions.sh
+
+START=61
+STOP=00
+
+USE_PROCD=1
+PROG=/usr/sbin/kresd
+CONFIGFILE=/tmp/kresd/kresd.config
+ROOTKEYFILE=/etc/knot-resolver/root.keys
+DEFAULT_RUNDIR=/tmp/kresd
+USERNAME=kresd
+GROUP=kresd
+
+service_triggers() {
+	procd_add_reload_trigger "dhcp"
+}
+
+init_header() {
+	echo "--Automatically generated file; DO NOT EDIT" > $CONFIGFILE
```

Comment:
This entire function can be condensed to a single file write.

From shellcheck: ^-- SC2129: Consider using { cmd1; cmd2; } >> file instead of individual redirects.